qrisk/stats.py

```diff
@@ -472,25 +508,30 @@ def information_ratio(returns, factor_returns):
     See https://en.wikipedia.org/wiki/information_ratio for more details.
 
     """
+
     active_return = returns - factor_returns
-    tracking_error = np.std(active_return, ddof=1)
```

Comment:
I would stick with the `np` prefix, makes it clear to everyone reading the code which one is being used.